clang/test/CheckedCRewriter/ptrtoconstarr.c

```diff
@@ -0,0 +1,12 @@
+// RUN: cconv-standalone -addcr -alltypes %s -- | FileCheck -match-full-lines %s
+// RUN: cconv-standalone --addcr --alltypes %s -- | %clang_cc1  -fcheckedc-extension -x c -
```

Comment:
Two requests:
(a) Can we make this test more expansive? The `int (*coef)[10]` case is the simplest possible. What if the array contains function pointers, per an earlier issue report? What about also typedefs and/or structs? Can it be an array of arrays?
(b) Can you add the bells and whistles of the other tests: Run with and without `-alltypes`, and make sure that the rewritten output can be passed through 3C and still yield reasonable output. Have a look at the preamble to the auto-gen tests.

Reply:
I don't see function pointers in the test below; pointer to an array of FPs ?